src/main/app/common/calculateInterest.ts

```diff
@@ -16,14 +16,9 @@ function calculateRate (interest: Interest): number {
   }
 }
 
-function calculateNumberOfDays (date: Moment): number {
-  const today = MomentFactory.currentDateTime()
-  return today.diff(date, 'days')
-}
-
-export function calculateInterest (amount: number, interest: Interest, interestDate: Moment) {
+export function calculateInterest (amount: number, interest: Interest, interestFromDate: Moment, interestToDate: Moment = MomentFactory.currentDateTime()) {
```

Comment:
Please also add one test scenario for case where `interestToDate` is set to something different then today.